.github/workflows/reusable_build_docker.yaml

```diff
@@ -0,0 +1,111 @@
+# This is a reusable workflow used by master and release CI
+on:
+  workflow_call:
+    inputs:
+      arch:
+        description: x86_64 or aarch64
+        required: true
+        type: string
+      bucket:
```

Comment:
I would also change the variable name since calling it "bucket" is a bit error prone, someone editing this workflow in the future might assume it's the bucket name!

Reply:
Yep, perhaps bucket_suffix? :)

Reply:
or just `suffix` would suffice (eheh)